src/components/Pool.vue

```diff
@@ -99,6 +99,11 @@ export default class Pool extends Mixins(TranslationMixin, LoadingMixin, NumberF
     return isWalletConnected()
   }
 
+  getAssetSymbol (address): string {
+    const asset = this.assets.find(a => a.address === address)
+    return asset ? asset.symbol : 'Unknown asset'
```

Comment:
Should this phrase 'Unknown asset' be translated, or it's not visible for users?

Reply:
I guess that we should add it to translation strings